pom.xml

```diff
@@ -479,6 +479,13 @@ flexible messaging model and an intuitive client API.</description>
         <artifactId>aspectjweaver</artifactId>
         <version>${aspectj.version}</version>
       </dependency>
+      
+	  <!-- TODO: should we callout BSD license: https://github.com/electronicarts/ea-agent-loader -->
```

Comment:
I think it should only be added to the `NOTICE` file that gets included in the binary distribution, not in the sources one, because there we're not including it.

I did some initial work in collecting the list of dependencies and the licenses to adjust the LICENSE and NOTICE files for both bin and src tgzs. You can leave it out for now, and remove the TODO comment. We'll fix it along with all the other dependencies.